packages/migrate/migrations/self-closing-tags/migrate.spec.js

```diff
@@ -0,0 +1,30 @@
+import { assert, test } from 'vitest';
+import { remove_self_closing_tags } from './migrate.js';
+
+/** @type {Record<string, string>} */
+const tests = {
+	'<div/>': '<div></div>',
+	'<div />': '<div></div>',
+	'<custom-element />': '<custom-element></custom-element>',
+	'<div class="foo"/>': '<div class="foo"></div>',
+	'<div class="foo" />': '<div class="foo"></div>',
+	'\t<div\n\t\tonclick={blah}\n\t/>': '\t<div\n\t\tonclick={blah}\n\t></div>',
+	'<foo-bar/>': '<foo-bar></foo-bar>',
+	'<link/>': '<link/>',
+	'<link />': '<link />',
+	'<svg><g /></svg>': '<svg><g /></svg>',
+	'<slot />': '<slot />',
+	'<svelte:options customElement="my-element" /><slot />':
+		'<svelte:options customElement="my-element" /><slot></slot>',
+	'<svelte:options namespace="foreign" /><foo />': '<svelte:options namespace="foreign" /><foo />',
+	'<script>console.log("<div />")</script>': '<script>console.log("<div />")</script>',
+	'<script lang="ts">let a: string = ""</script><div />':
+		'<script lang="ts">let a: string = ""</script><div></div>'
```

Comment:
ideally this would work on Svelte 5 codebases too (which means having `svelte@next` as a dependency, and using walk from either `estree-walker` or `zimmerframe` instead of `svelte/compiler`)

```suggestion
		'<script lang="ts">let a: string = ""</script><div></div>',
	'{#snippet foo()}<div />{/snippet}': '{#snippet foo()}<div></div>{/snippet}'
```

Reply:
This should work in Svelte 5 code bases, too, because when you do `npx ..` aren't you storing the whole dependenies in a isolated folder somewhere on your computer? If not then yes we can switch to zimmerframe.

Reply:
Using the local Svelte installation now, tested it successfully on the Svelte 5 repo